pkg/util/push/otlp.go

```diff
@@ -37,31 +41,22 @@ func OTLPHandler(sourceIPs *middleware.SourceIPExtractor, push Func) http.Handle
 			return
 		}
 
-		promConverter := prometheusremotewrite.NewPrometheusConverter()
-		setting := prometheusremotewrite.Settings{
-			AddMetricSuffixes: true,
-			DisableTargetInfo: true,
```

Comment:
We need to expose `DisableTargetInfo` as a flag. If people choose to convert all attributes then it makes sense to not have target info metric